lib/TopSitesFeed.jsm

```diff
@@ -70,10 +71,13 @@ this.TopSitesFeed = class TopSitesFeed {
     PageThumbs.addExpirationFilter(this);
   }
 
-  init() {
+  async init() {
     // If the feed was previously disabled PREFS_INITIAL_VALUES was never received
     this.refreshDefaults(this.store.getState().Prefs.values[DEFAULT_SITES_PREF]);
     this._storage = this.store.dbStorage.getDbTable("sectionPrefs");
+    if (this.store.getState().Prefs.values[SEARCH_SHORTCUTS_EXPERIMENT]) {
```

Comment:
I don't think it's sufficient to only run this on `init`, this also need needs to be run when the pref updates etc. (e.g. when shield sets the experiment pref, particularly if we decide to ship to release with this default-off)

Perhaps this should be checked on every refresh i.e. in `getLinksWithDefaults`?